apps/estuary/config/prod.exs

```diff
@@ -0,0 +1,4 @@
+use Mix.Config
+
+# Do not print debug messages in production
```

Comment:
Are these comments helpful? (this and the one in the test config)

Reply:
Sorry, I thought I had removed this earlier. It's gone now.